egui/src/containers/scroll_area.rs

```diff
@@ -155,6 +171,27 @@ impl Prepared {
 
         let content_size = content_ui.min_size();
 
+        let scroll_target = content_ui.ctx().frame_state().scroll_target();
+        if let Some(scroll_target) = scroll_target {
+            let center_ratio = content_ui.ctx().frame_state().scroll_target_center_ratio();
+            let height_offset = content_ui.clip_rect().height() * center_ratio;
+            let top = content_ui.min_rect().top();
+            let offset_y = scroll_target - top - height_offset;
```

Comment:
```suggestion
            let offset_y = scroll_target - lerp(content_ui.clip_rect().y_range(), center_ratio);
```

Reply:
This does not work, as the `clip_rect` is always the same, but I updated the code to be more easy to understand. I hope.